packages/mco/utils/acm-search-queries.ts

```diff
@@ -65,6 +65,30 @@ export const queryNamespacesUsingCluster = (
   query: searchFilterQuery,
 });
 
+export const queryStorageClassesUsingCluster = (
+  clusterName: string | string[]
```

Comment:
```suggestion
  clusterNames: string[]
```

while passing, we can convertthe  string to a list

Reply:
Ack, changed